lib/statsd/instrument.rb

```diff
@@ -448,11 +448,16 @@ def parse_options(value, metric_options)
 
   # Instantiates a metric, and sends it to the backend for further processing.
   # @param options (see StatsD::Instrument::Metric#initialize)
-  # @return [StatsD::Instrument::Metric] The meric that was sent to the backend.
+  # @return [StatsD::Instrument::Metric] The metric that was sent to the backend.
   def collect_metric(type, name, value, metric_options)
     value, metric_options = parse_options(value, metric_options)
-
-    options = hash_argument(metric_options).merge(type: type, name: name, value: value)
+    metric_options = hash_argument(metric_options)
+    options = metric_options.merge(
+      type: type,
+      name: name,
+      value: value,
+      prefix: metric_options[:prefix]
```

Comment:
what does this line actually do? I'm reading this as `metric_options.merge(prefix: metric_options[:prefix])`, which doesn't look like it's doing anything, is it?

Reply:
Ah, you are right I goofed